remix-solidity/src/compiler/compiler-worker.ts

```diff
@@ -2,8 +2,8 @@
 
 var solc = require('solc/wrapper')
 
-var compileJSON = function () { return '' }
-var missingInputs = []
+var compileJSON: any|null = (input) => { return '' }
+var missingInputs: any = []
```

Comment:
You know that it's an array at least: `any[]`.

Reply:
updated